src/components/Nav/Nav.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import { GiHamburgerMenu } from 'react-icons/gi';
+import { GoSearch } from 'react-icons/go';
+import { BsHandbag, BsInstagram, BsYoutube } from 'react-icons/bs';
+import Logo from './Logo';
+import SearchBox from './SearchBox';
+import RightBoxNav from './RightBoxNav';
+import './Nav.scss';
+
+const Nav = () => {
+  const [isToggle, setToggle] = useState(true);
+  const [isSearch, setIsSearch] = useState(false);
+
+  const handleClick = () => {
+    setToggle(!isToggle);
+  };
+
+  const handleSearchClick = () => {
+    setIsSearch(!isSearch);
+  };
+
+  return (
+    <>
+      <nav className="nav">
+        <ul className="navText">
+          <Logo />
+          <div className="navTextList">
+            ABOUT
+            <div className="navTextListInfo">
+              {ABOUT.map(aboutItem => (
+                <li key={aboutItem.id}>{aboutItem.listName}</li>
+              ))}
```

Comment:
```suggestion
              {ABOUT.map(aboutItem => (
                <li key={aboutItem.id}>{aboutItem.listName}</li>
              ))}
```
item, element 등의 접미사는 지양해주세요!

실제로 aboutItem, aboutElement, aboutInfo, aboutData 등 이런 접미사를 붙였을 때 각각 무엇이 다른지 판단하기 어려워서 지양합니다!

Reply:
```suggestion
              {ABOUT.map(({id, listName}) => (
                <li key={id}>{listName}</li>
              ))}
```
매개변수자리에서 바로 구조분해할당 해서 사용할 수 있겠네요